docs/evaluation/how_to_guides/datasets/manage_datasets_in_application.mdx

```diff
@@ -13,70 +13,96 @@ Before diving into this content, it might be helpful to read the following:
+The first way is to multi-select runs from the runs table:
 
-:::
+![](./static/multiselect_add_to_dataset.png)
+
+The second is to navigate to the run details page and click `Add to -> Dataset` on the top right corner:
+
+![](./static/add_to..dataset.png)
 
-![Add to Dataset](./static/add_to_dataset.png)
+When you select the proper dataset from the run details page (this one using transformations to optimize for collecting LLM runs), 
+a modal will pop up letting you know if any transformations were applied / if schema validation failed. You can then optionally edit 
+the run before adding it to the dataset. 
 
-From there, we select the dataset to organize it in and update the ground truth output values if necessary.
+![](./static/confirmation.png)
 
-![Modify example](./static/modify_example.png)
+### Set up an automation to add a subset of all runs in a tracing project to a dataset
 
-## Upload a CSV file to create a dataset
+You can use [run rules](../../../observability/how_to_guides/monitoring/rules) to automatically add traces to a dataset based on certain conditions. For example, you could add all traces that have a certain tag to a dataset.
 
-The easiest way to create a dataset from your own data is by clicking the 'upload a CSV dataset' button on the home page or in the top right-hand corner of the 'Datasets & Testing' page.
+Note you can also set up rules to add runs that meet criteria to an annotation queue, and then follow the guide below for adding from annotation queues.
 
-![Upload CSV](./static/create_dataset_csv.png)
+### Add run(s) from an annotation queue
 
-Select a name and description for the dataset, and then confirm that the inferred input and output columns are correct.
+Annotation queues are configured optionally with a default dataset, though they allow you to add runs to any dataset 
+by simply pressing the dataset switcher on the bottom of the screen. Once you select the right dataset, you click `Add to Dataset` 
+or hit the hot key `D` to add the run to it. 
 
-![Confirm Columns](./static/select_columns.png)
+![](./static/add_to_dataset_from_aq.png)
 
-## Generate synthetic examples
+As with adding directly from the run details page, any modifications you make to the run in your annotation queue will carry over to 
+the dataset, and all metadata associated with the run will also be copied.
 
-For a dataset with a specified schema, you can generate synthetic examples to enhance your dataset.
+### Add runs directly via the Datasets UI
+
+Go to the Datasets & Experiments UI, click the `Examples` tab, and `+ Example`. This will let you either define 
+JSON inline
+
+### Add synthetic examples created by an LLM via the Datasets UI
+
+If you have a schema defined on your dataset, when clicking `+ Example` as in the previous section, you can decide to generate examples instead of manually input them. 
```

Comment:
```suggestion
If you have a schema defined on your dataset, when you click `+ Example` you'll see an option to `Generate examples`. This will use an LLM to create synthetic examples.
```